redwood-lazylayout-widget/src/commonMain/kotlin/app/cash/redwood/lazylayout/widget/LazyListUpdateProcessor.kt

```diff
@@ -458,9 +465,13 @@ public abstract class LazyListUpdateProcessor<V : Any, W : Any> {
     public val isBound: Boolean
       get() = view != null
 
-    internal fun bind(view: V) {
+    public fun bind(view: V) {
       require(this.view == null) { "already bound" }
 
+      if (isPlaceholder && this.content == null) {
+        this.content = processor.takePlaceholder()
+      }
+
```

Comment:
Without the structural change to a binding's state, should we add `require(this.content != null)` for the non-placeholder case?

Reply:
Great idea. Lemme do that in the above-mentioned follow-up.